spec.md

```diff
@@ -289,6 +290,34 @@ encapsulated within the `data` attribute.
 * Constraints:
   * OPTIONAL
 
+# Minimum Supported Event Size
+
+In order to increase interoperability, all CloudEvent consumers SHOULD accept
+events that follow these rules:
+
+* The number of attributes does not exceed 100. Each index of a `Map` is
+  considered an attribute.
+* All attribute names are 20 characters or less long.
+* All indexes of `Maps` are 20 characters or less long.
+* All `Binary` attributes do not exceed a size of 1KB.
```

Comment:
Does this include `data` as a `Binary` attribute, or only context attributes?

Reply:
`data` is an `Any` attribute, and the rule for `Any` attributes states that it does not apply to `data`.

Maybe it is easier to read if I move that line (305) up.